dll/win32/shell32/CDefView.cpp

```diff
@@ -2807,8 +2804,9 @@ HRESULT STDMETHODCALLTYPE CDefView::AutoArrange()
 
 HRESULT STDMETHODCALLTYPE CDefView::AddObject(PITEMID_CHILD pidl, UINT *item)
 {
-    FIXME("(%p)->(%p %p) stub\n", this, pidl, item);
-    return E_NOTIMPL;
+    TRACE("(%p)->(%p %p)\n", this, pidl, item);
+    *item = LV_AddItem(pidl);
+    return *item >= 0 ? S_OK : E_OUTOFMEMORY;
```

Comment:
This needs to be checked while it's still an `int` -- `*item` is an `UINT` so it's always `>=0`